tailwind.config.js

```diff
@@ -1,11 +1,35 @@
+const colors = require('tailwindcss/colors')
+const path = require('path');
+const distPath = path.resolve(__dirname, 'src');
+
 module.exports = {
   purge: [],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    borderColor: theme => ({
+      ...theme('colors'),
+      'rust': "#e56034"
+    }),
+    minWidth: {
+      'project-card-mobile': "320px",
+      'project-card': "420px"
+    },
+    extend: {
+      colors:{
+        header: "#E8C64E"
+      },
+      fontFamily:{
+        playfair: ["Playfair Display"],
+        lato: ["Lato"],
+      },
```

Comment:
I suggest using semantic names for this, instead of using the font names itself.
Also, don't forget the font fallback.

```suggestion
      fontFamily:{
        heading: ['Playfair Display', 'sans-serif'],
        body: ['Lato', 'sans-serif'],
      },
```

Reply:
In this case, I do not agree because the problem is that Lato not only is used in the body, in some cases is used to be the font of titles, and in other cases, Playfair is used in subtitles. 
It's complicated to use a semantic name to the fonts.

Reply:
It doesn't matter, actually design is more dynamic than it seems. The goal is to have a semantic name but only place I've seen some strictness on usage is in a Word document 😂 .

Im just suggesting to use semantic names over the name of the font-family as Tailwind is intended for standardization, instead of having an arbitrary set of font names all around the code you stick to "aliases" to these names.